apps/server/src/modules/learnroom/service/common-cartridge-export.service.spec.ts

```diff
@@ -44,183 +42,152 @@ describe('CommonCartridgeExportService', () => {
 					provide: TaskService,
 					useValue: createMock<TaskService>(),
 				},
+				{
+					provide: ConfigService,
+					useValue: createMock<ConfigService<LearnroomConfig, true>>(),
+				},
 			],
 		}).compile();
-		courseExportService = module.get(CommonCartridgeExportService);
+		sut = module.get(CommonCartridgeExportService);
 		courseServiceMock = module.get(CourseService);
 		lessonServiceMock = module.get(LessonService);
 		taskServiceMock = module.get(TaskService);
-		course = courseFactory.teachersWithId(2).buildWithId();
-		lessons = lessonFactory.buildListWithId(5, {
-			contents: [
-				{
-					component: ComponentType.TEXT,
-					title: 'Text',
-					content: {
-						text: 'text',
-					},
-				} as ComponentProperties,
-				{
-					component: ComponentType.ETHERPAD,
-					title: 'Etherpad',
-					content: {
-						url: 'url',
-					},
-				} as ComponentProperties,
-				{
-					component: ComponentType.GEOGEBRA,
-					title: 'Geogebra',
-					content: {
-						materialId: 'materialId',
-					},
-				} as ComponentProperties,
-				{} as ComponentProperties,
-			],
-		});
-		tasks = taskFactory.buildListWithId(5, {
-			name: 'Task of a lesson',
-			lesson: lessons[0],
-		});
+		configServiceMock = module.get(ConfigService);
 	});
 
 	afterAll(async () => {
 		await module.close();
 	});
 
 	describe('exportCourse', () => {
-		const setupExport = async (version: CommonCartridgeVersion) => {
+		const setup = async (version: CommonCartridgeVersion) => {
```

Comment:
Move setup to when descibes. But you can outsource the creation and make the param CommonCartridgeVersion.V_1_1_0 or 1_3_0 inside of the setup instead in the "it"`s.

Reply:
Done.